Sources/Testing/ABI/EntryPoints/EntryPoint.swift

```diff
@@ -275,6 +275,9 @@ public struct __CommandLineArguments_v0: Sendable {
 
   /// The value of the `--repeat-until` argument.
   public var repeatUntil: String?
+
+  /// The value of the `--experimental-attachments-path` argument.
+  public var experimentalAttachmentsPath: String?
```

Comment:
Is there a reason this property doesn't need `@_spi(Experimental)` too?

Reply:
`__CommandLineArguments_v0` is unstable.